pkg/tasks/snapshot_helper.go

```diff
@@ -55,7 +55,17 @@ func (sh *SnapshotHelper) Run(versionHref string) error {
 		}
 	}
 
-	distHref, distPath, addedContentGuard, err := sh.createDistribution(publicationHref, sh.repo.UUID, *sh.payload.GetSnapshotIdent())
+	distPath := fmt.Sprintf("%v/%v", sh.repo.UUID, *sh.payload.GetSnapshotIdent())
+	helper := helpers.NewPulpDistributionHelper(sh.ctx, sh.pulpClient)
+
+	distHref, addedContentGuard, err := helper.CreateOrUpdateDistribution(sh.orgId, *sh.payload.GetSnapshotIdent(), distPath, publicationHref)
+	if err != nil {
+		return err
+	}
+
+	latestPathIdent := fmt.Sprintf("%v/%v", sh.repo.UUID, "latest")
+
+	_, _, err = helper.CreateOrUpdateDistribution(sh.orgId, sh.repo.UUID, latestPathIdent, publicationHref)
 	if err != nil {
 		return err
 	}
```

Comment:
I noticed that for a previously existing repo, a distribution isn't created because the snapshot task returns before reaching this point (if the snapshot has no changes). Would we need to maybe run a job that creates the distribution for the existing repos?

Reply:
Yeah, I was worried about this as well. 
I'll look into this.

Reply:
Would this be better done via a cron job?

Reply:
Sorta, I was thinking a one-time job that runs after this merges. But a cron job is reoccurring, I don't think we'd need it to be reoccurring?

Reply:
As mentioned in standup, we'll grab another card for this.
Thank you for the callout @rverdile!